vendor/github.com/russross/blackfriday/LICENSE.txt

```diff
@@ -1,29 +0,0 @@
-Blackfriday is distributed under the Simplified BSD License:
```

Comment:
The problem is that `glide-vc` is not taking test dependencies into account.

We also need to stop committing broken garbage into the Makefile. None of this is making this easier.